Document.cs

```diff
@@ -141,7 +207,7 @@ public bool AnchorAsName
 			// Not available before until 18 Jun 2008
 			get
 			{
-				if (this.ReleaseDate < new DateTime(2008, 6, 18))
+				if (MajorVersion < 4)
```

Comment:
Why do we need all this? Don't we expect to release an appropriate version of libtidy as part of our package?